SessionUIKit/Components/SwiftUI/SessionSearchBar.swift

```diff
@@ -0,0 +1,77 @@
+// Copyright © 2024 Rangeproof Pty Ltd. All rights reserved.
+
+import SwiftUI
+
+public struct SessionSearchBar: View {
+    @Binding var searchText: String
+    
+    let cancelAction: () -> Void
+    
+    let height: CGFloat = 40
+    let cornerRadius: CGFloat = 7
+    
+    public init(searchText: Binding<String>, cancelAction: @escaping () -> Void) {
+        self._searchText = searchText
+        self.cancelAction = cancelAction
+    }
+    
+    public var body: some View {
+        HStack(
+            alignment: .center,
+            spacing: 0
+        ) {
+            HStack(
+                alignment: .center,
+                spacing: Values.verySmallSpacing
+            ) {
+                Image(systemName: "magnifyingglass")
+                    .font(.system(size: Values.smallFontSize))
+                    .foregroundColor(themeColor: .textSecondary)
+                    .padding(.horizontal, Values.smallSpacing)
+                
+                ZStack(alignment: .leading) {
+                    if searchText.isEmpty {
+                        Text("Search")
```

Comment:
Should this be localised?

Reply:
Yep, this will also be fixed with the Strings project